src/main/java/com/shapesecurity/salvation/data/Policy.java

```diff
@@ -144,6 +144,9 @@ private void optimise() {
                         .filter(x -> !(x instanceof HostSource))
                         // * remove network-schemes in source list that contains *
                         .filter(x -> !(x instanceof SchemeSource) || !((SchemeSource) x).matchesNetworkScheme())
+                        // remove 'unsafe-inline' if needed
```

Comment:
I would rather have this comment say something more like unsafeInlineWarningMessage.